### DISCUSSION TITLE
Can’t log into mobile app

### DISCUSSION THREAD

**Discussion by tflanagan** [community/community#15123] [Mobile] [answered]
Unable to load keyboard on mobile app when signing in.

![7E939A3E-69E1-4033-B8B0-6746FD8A7380](https://user-images.githubusercontent.com/5559820/163797582-7b8b4cfd-33d7-45ca-b5e7-8dc69160be13.jpeg)
![29A22E2C-EB0D-417B-98E9-8493765E2C30](https://user-images.githubusercontent.com/5559820/163797585-92a1147a-d981-4293-a638-6dfa887a3141.png)

**Comment by HenSquared** [comment, marked as answer]
Hi @tflanagan,
Thanks a bunch for reporting the issue.  We've noticed this issue before and believe it is an issue on iOS/iPadOS. I have filed a bug with Apple reporting this behavior. We also have an issue tracking this behavior. I will add this report to that issue and follow up with the team internally to figure out next steps to fix this. 

Here are some workarounds that may help unblock you for now. 

**Copy/Paste**
You may be able to paste your username and password into those fields.
1. Copy username/password from password manager or Notes app (or other app)
2. Paste into the fields.

**Backgrounding App**
I've had this workaround work sometimes. It is not 100%, but may be worth trying. 
1. While the field is selected (cursor is currently in the field, like in screenshot you shared)
2. Background GitHub by returning to home screen.
3. Come back to GitHub 
Keyboard may appear. 

**Force closing app**
I've had this workaround work sometimes. It is not 100%, but may be worth trying. 
1. Force close the app [with these instructions](https://support.apple.com/en-us/HT201330) 
2. Reopen the app and try again.
